core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -19,11 +19,9 @@ cluster_groups:
   - vsphere
   non_app_ci_arm:
   - arm01
-  non_app_ci_x86:
```

Comment:
[build-clusters][1] is max about this going away.  I dunno if that's a problem, or just that the test was not designed to handle this sort of pivot.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/26914/pull-ci-openshift-release-master-build-clusters/1502033600441225216

Reply:
Maybe avoiding this failure is what openshift/ci-tools#2718 is about.

Reply:
Correct.

Reply:
openshift/ci-tools#2718 landed.  I dunno if it's waiting on some auto-promoter to get tied in here, but in case it is not:

/test build-clusters

Reply:
Need to wait for https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/branch-ci-openshift-ci-tools-master-images/1502066390071250944